plugins/repository-s3/src/internalClusterTest/java/org/opensearch/repositories/s3/S3RepositoryThirdPartyTests.java

```diff
@@ -52,6 +55,21 @@
 
 public class S3RepositoryThirdPartyTests extends AbstractThirdPartyRepositoryTestCase {
 
+    @Override
+    @Before
+    @SuppressForbidden(reason = "Need to set system property here for AWS SDK v2")
+    public void setUp() throws Exception {
+        SocketAccess.doPrivileged(() -> System.setProperty("opensearch.path.conf", "config"));
+        super.setUp();
+    }
+
+    @Override
+    @SuppressForbidden(reason = "Need to reset system property here for AWS SDK v2")
+    public void tearDown() throws Exception {
+        SocketAccess.doPrivileged(() -> System.clearProperty("opensearch.path.conf"));
```

Comment:
I think it is safer to restore the previous value (we need to capture that) if it was set (fe from `build.gradle` as on option) or from Gradle command line

Reply:
I think this is a NoOp as the all the tests here are run in isolation. @dblock can you verify if we need to clear the property here?

Reply:
@raghuvanshraj  this is not correct, see please the Gradle docs here: https://docs.gradle.org/current/userguide/performance.html#execute_tests_in_parallel. The forked JVM will have its system property modified and visible for all other tests which are going to be run on it later on.